### PR TITLE
[RFC] vim-patch:7.4.831

### PR DESCRIPTION
**Pull request by jbradaric**
Problem:    When expanding  on the command line and encountering an
            error, the command is executed anyway.
Solution:   Bail out when an error is detected.

https://github.com/vim/vim/commit/3f188935ec4db5117c4a64cc3f71219175624745

### CODE REVIEW

File: src/nvim/path.c [hunk 1]
```diff
@@ -1246,13 +1251,11 @@ static int vim_backtick(char_u *p)
...
- * Currently only works when pat[] starts and ends with a `.
- * Returns number of file names found.
- */
-static int 
-expand_backtick (
+// Expand an item in `backticks` by executing it as a command.
+// Currently only works when pat[] starts and ends with a `.
+// Returns number of file names found, -1 if an error is encountered.
+static int
+expand_backtick(
```
Comment (cacplate) [inline src/nvim/path.c]: type, name and parameters on same line is better.

**Comment by watiko** [comment]
#4247 covered `vim-patch:7.4.831`

**Comment by Shougo** [comment]
Oh, it is duplicated...

**Comment by jbradaric** [comment]
I guess I forgot to check whether this was already done.

**Pull request closed without merging**